libensemble/comms/comms.py

```diff
@@ -208,10 +209,13 @@ def result(self, timeout=None):
         return self._result
 
     @staticmethod
-    def _qcomm_main(comm, main, *args, **kwargs):
+    def _qcomm_main(comm, main, *fargs, **kwargs):
         """Main routine -- handles return values and exceptions."""
         try:
-            _result = main(comm, *args, **kwargs)
+            if not kwargs.get("ufunc"):
```

Comment:
Call something like user_func to not confuse with ufunc for universal functions.